main.js

```diff
@@ -0,0 +1,11 @@
+const hamburger = document.getElementById('pop');
+const closebutton = document.querySelector('.close');
+const popupWindow = document.getElementById('about');
+
+hamburger.addEventListener('click', () => {
+  popupWindow.classList.toggle('hidden');
+});
+
+closebutton.addEventListener('click', () => {
+  popupWindow.classList.toggle('hidden');
+});
```

Comment:
![image](https://user-images.githubusercontent.com/60612329/164735854-f00f15bc-25fb-4f2f-9b01-f4b508cf081a.png)
 as seen above the following needs to be addressed 

- when the menu is clicked the icon should be hidden and should only display when the popup is closed 😄